go.mod

```diff
@@ -11,6 +11,6 @@ require (
 	github.com/polydawn/refmt v0.0.0-20201211092308-30ac6d18308e
 	github.com/smartystreets/goconvey v1.6.4 // indirect
 	github.com/warpfork/go-testmark v0.3.0
-	github.com/warpfork/go-wish v0.0.0-20200122115046-b9ea61034e4a
+	github.com/warpfork/go-wish v0.0.0-20200122115046-b9ea61034e4a // indirect
```

Comment:
I am not sure where this is coming from; this is what the dependency graph tells me:

```
$ go mod graph | grep go-wish
github.com/ipld/go-ipld-prime github.com/warpfork/go-wish@v0.0.0-20200122115046-b9ea61034e4a
```

Reply:
```
$ go mod why -m github.com/warpfork/go-wish
# github.com/warpfork/go-wish
github.com/ipld/go-ipld-prime/codec/dagcbor
github.com/polydawn/refmt/cbor
github.com/polydawn/refmt/cbor.test
github.com/warpfork/go-wish
```

Not sure why go mod graph shows that edge, to be honest.

Reply:
Also, the reason why the indirect line stays is because refmt is not a Go module yet, so then its dependencies have to be tracked here.

Reply:
FYI @warpfork if you're interested in actually dropping go-wish from the dependency graph. Most ipld-prime users shouldn't need to download nor link go-wish though, if they are only using the ipld-prime library.

Reply:
> Also, the reason why the indirect line stays is because refmt is not a Go module yet, so then its dependencies have to be tracked here.

Bingo; that explains the `go mod graph` output thanks @mvdan  🍻